daemon/defaults/defaults.go

```diff
@@ -19,7 +19,7 @@ const (
 	RuntimePath = "/var/run/cilium"
 
 	// RuntimePathRights are the default access rights of the RuntimePath directory
-	RuntimePathRights = 0770
+	RuntimePathRights = 0775
```

Comment:
Should we change the cilium socket to `/var/run/cilium.sock` instead of `/var/run/cilium/cilium.sock`?